.github/CODEOWNERS

```diff
@@ -327,6 +327,7 @@
 /src/plugins/interactive_setup/ @elastic/kibana-security
 /test/interactive_setup_api_integration/ @elastic/kibana-security
 /test/interactive_setup_functional/ @elastic/kibana-security
+/test/plugin_functional/test_suites/core_plugins/rendering.ts @elastic/kibana-security
```

Comment:
Added the integration test to my team in CODEOWNERS so we get pinged when this test is changed :+1: